src/category_theory/endofunctor/algebra.lean

```diff
@@ -394,22 +390,25 @@ def coalgebra.to_algebra_of (adj : F ⊣ G) : coalgebra G ⥤ algebra F :=
 adjoint and going back is isomorphic to the identity functor. -/
 def alg_coalg_equiv.unit_iso (adj : F ⊣ G) :
   𝟭 (algebra F) ≅ (algebra.to_coalgebra_of adj) ⋙ (coalgebra.to_algebra_of adj) :=
-{ hom := { app := λ A, { f := (𝟙 A.1),
-                           h' := begin dsimp, rw [F.map_id, category.id_comp, category.comp_id],
-                                      apply (adj.hom_equiv _ _).left_inv A.str, end },
-                         naturality' := λ A₁ A₂ f,
-                           begin dsimp, ext1, dsimp, rw[category.id_comp, category.comp_id],
-                                 refl end, },
-                inv := { app := λ A, { f := (𝟙 A.1),
-                          h' := begin dsimp, erw [F.map_id, category.id_comp, category.comp_id],
-                                      apply ((adj.hom_equiv _ _).left_inv A.str).symm end },
-                         naturality' := λ A₁ A₂ f,
-                           begin dsimp, ext1, dsimp, erw [category.comp_id, category.id_comp],
-                                 refl, end },
-                hom_inv_id' := begin dsimp, ext1, dsimp, ext1, dsimp, ext1, dsimp,
+{ hom :=
+    { app := λ A,
+        { f := (𝟙 A.1),
+          h' := by { erw [F.map_id, category.id_comp, category.comp_id],
+                       apply (adj.hom_equiv _ _).left_inv A.str } },
+      naturality' := λ A₁ A₂ f,
+        by { ext1, dsimp, erw [category.id_comp, category.comp_id], refl } },
+  inv :=
+    { app := λ A,
+        { f := (𝟙 A.1),
+          h' := by { erw [F.map_id, category.id_comp, category.comp_id],
+                       apply ((adj.hom_equiv _ _).left_inv A.str).symm } },
+      naturality' := λ A₁ A₂ f,
+        by { ext1, dsimp, erw [category.comp_id, category.id_comp], refl } },
+  hom_inv_id' := begin dsimp, ext1, dsimp, ext1, dsimp, ext1, dsimp,
```

Comment:
Possibly these lines could be golfed a bit, but I have run out of time. I'll have another look today or tomorrow. Thanks a lot for the PR!

Reply:
I golfed the last two lines too (Also I realised the indentation should be fixed to the following):
```
def alg_coalg_equiv.unit_iso (adj : F ⊣ G) :
  𝟭 (algebra F) ≅ (algebra.to_coalgebra_of adj) ⋙ (coalgebra.to_algebra_of adj) :=
{ hom :=
  { app := λ A,
    { f := (𝟙 A.1),
      h' := by { erw [F.map_id, category.id_comp, category.comp_id],
                   apply (adj.hom_equiv _ _).left_inv A.str } },
    naturality' := λ A₁ A₂ f, by { ext1, dsimp, erw [category.id_comp, category.comp_id], refl } },
  inv :=
  { app := λ A,
      { f := (𝟙 A.1),
        h' := by { erw [F.map_id, category.id_comp, category.comp_id],
                     apply ((adj.hom_equiv _ _).left_inv A.str).symm } },
    naturality' := λ A₁ A₂ f,
      by { ext1, dsimp, erw [category.comp_id, category.id_comp], refl } },
  hom_inv_id' := by { ext, exact category.comp_id _ },
  inv_hom_id' := by { ext, exact category.comp_id _ } }
```

Reply:
By I'm sorry that I just edited the file, I now realize that I probably should have done some kind of github trick with "review". This is the first time I've actually reviewed something

Reply:
No worries. Feel free to just push these changes. Thanks a lot for your time.